conf/prod.routes

```diff
@@ -1,5 +1,5 @@
 # Add all the application routes to the app.routes file
-->         /help-to-save-stride         app.Routes
+->         /digitally-excluded          app.Routes
```

Comment:
its better to have `/help-to-save` prefix so it becomes `/help-to-save/digitally-excluded` ? 
otherwise url becomes `www.qa.tax.service.gov.uk/digitally-excluded` which doesn't seem right .
we can have `www.qa.tax.service.gov.uk//help-to-save/digitally-excluded`
Also if you change the root url here, you should also change the mdtp routing config it here 
`https://github.com/hmrc/mdtp-frontend-routes/blob/c6d3d7bcc3a1ace6bcedd8370b0760086f1b13b9/staging/frontend-proxy-application-rules.conf#L2644-L2660`